examples/lunar_lander.py

```diff
@@ -145,11 +145,12 @@ def create_optimizer(seed, n_emitters, sigma0, batch_size):
     env = gym.make("LunarLander-v2")
     action_dim = env.action_space.n
     obs_dim = env.observation_space.shape[0]
-
+    initial_model = np.zeros((action_dim, obs_dim))
     archive = GridArchive(
-        [50, 50],  # 50 cells in each dimension.
-        [(-1.0, 1.0), (-3.0, 0.0)],  # (-1, 1) for x-pos and (-3, 0) for y-vel.
-        seed=seed,
+        solution_dim=len(initial_model.flatten()),
```

Comment:
Use `ravel()` instead of `flatten()` since `flatten` makes a copy but `ravel` does not make a copy unless needed.

Reply:
Actually, just use `initial_model.size` instead of `len(...)` https://numpy.org/doc/stable/reference/generated/numpy.ndarray.size.html

Reply:
okie (I just used flatten cuz it was used below)